borgcron.sh

```diff
@@ -4,14 +4,8 @@
 # LICENSE: MIT license, see LICENSE.md
 #
 
-
-###########
-# Default settings (see also borgcron_worker.sh)
-###########
-CONFIG_DIR='config'
-
-
-
+# dir where config files are stored
+CONFIG_DIR="$( dirname "$0" )/config"
```

Comment:
Good question. Because when doing it in a relative fashion(`./config`) this is always relative to the working dir where the script is started. That could be troublesome, especially with cron or so… (No idea what working dir they use.) This explicitly uses the dir, where the script is stored.